web/public/js/controllers/user-profile-controller.js

```diff
@@ -173,6 +173,16 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
     return cdProgrammingLanguagesService.get();
   };
 
+
+  $scope.imageByLanguage = function(language){
+    cdProgrammingLanguagesService.getByLanguage(language,
+      function (responseData) {
+        $scope.profile.programmingLanguagesImageUrl.push(responseData[0]);
+    }, function () {
+      console.error("error when get data")
```

Comment:
(cian, we don't have logger on the front-end, only the back :p)
But yeah, we avoid having console handling for errors, prefer bootbox to display an alert (we'll come to a generic error handler soon, that's on our github project :))